app/views/event_instances/_hangout_button.html.erb

```diff
@@ -8,8 +9,32 @@
     'callbackUrl' => hangout_url('id').gsub(/id$/, '')
   }) %>
```

Comment:
I would suggest moving this out to a helper class. It feels like too much logic for the view. It was a cornercase before and it definitely now with this if condition.